content/docs/04-clusters/02-data-center/03-vmware.md

```diff
@@ -629,23 +627,170 @@ You can set up the PCG as a single- or three-node cluster based on your requirem
+| **Pod CIDR**|Enter the CIDR pool that will be used to assign IP addresses to pods in the PCG cluster. The pod IP addresses should be unique and not overlap with any machine IPs in the environment.|
+| **Service IP Range**|Enter the IP address range that will be used to assign IP addresses to services in the PCG cluster. The service IP addresses should be unique and not overlap with any machine IPs in the environment.|
+
+#### VMware vSphere Account Information
+
+|**Parameter**                            | **Description**|
+|-----------------------------------------|----------------|
+|**vSphere Endpoint** | vSphere endpoint: FQDN or IP address, without a scheme (i.e. 'https://'). <br />e.g. vcenter.mycompany.com|
+|**vSphere Username**  | vSphere account username|
+|**vSphere Password** | vSphere account password|
+|**Allow Insecure Connection (Bypass x509 Verification)** |Enter 'y' if using a vSphere instance with self-signed TLS certificates. Otherwise, enter 'n'.|
+
+#### VMware vSphere Private Cloud Gateway cluster configuration
+
+1. Select and/or specify values for the following properties:
+  * Datacenter
+  * Folder
+  * Fault Domain(s). Configure one or more fault domains by selecting values for the following properties:
+    * Cluster
+    * Network
+    * Resource Pool
+    * Storage Type (Datastore or VM Storage Policy)
+  * NTP server(s)
+  * SSH Public Key(s)
+  * PCG cluster size: **1** or **3** nodes (HA)
+
+2. Specify IP Pool configuration
+  * Placement Type (Static or DDNS). With static placement, an IP pool is created and VMs are assigned IPs from that pool. With DDNS, VMs are assigned IPs via DNS.
+
+##### Static Placement Configuration
+|**Parameter**                            | **Description**|
+|-----------------------------------------|----------------|
+| **IP Start range** | Enter the first address in the PCG IP pool range.|
+| **IP End range** | Enter the last address in the PCG IP pool range.|
+| **Network Prefix** | Enter the network prefix for the IP pool range, e.g. 18. Valid values are in [0, 32].|
+| **Gateway IP Address** | .|
+| **Name server(s)** | Comma separated list of DNS name server IP addresses.|
+| **Name server search suffixes (optional)** | Comma separated list of DNS search domains.|
+
+##### DDNS Placement Configuration
+|**Parameter**                            | **Description**|
+|-----------------------------------------|----------------|
+| **Search domain(s)** | Comma separated list of DNS search domains.|
+
+3. Specify cluster boot configuration
+
+|**Parameter**                            | **Description**|
+|-----------------------------------------|----------------|
+| **Patch OS on boot** | Whether or not to patch the OS of the PCG hosts on first boot.|
+| **Reboot node(s) once OS patch is applied (if required)** | Whether or not to reboot PCG nodes after OS patches are complete. Only applies if Patch OS on boot is enabled.|
+
```

Comment:
@TylerGillson For lines 735 and 736, you can use the wording suggested in the OpenStack md file.